static/templates/message_edit_history.handlebars

```diff
@@ -0,0 +1,15 @@
+{{! Client-side Mustache template for viewing message edit history.}}
+
+<div class="date_row"><span id="timerender2">Earliest</span></div>
+{{#each edited_messages}}
+<div class="">
+    <div class="messagebox-border">
+      <div class="messagebox-content">
+        <div class="message_top_line"><span class="message_time">{{ timestamp }}</span></div>
+        <div class="message_edit_content">{{{ prev_rendered_content }}}</div>
+        <div class="message_author"><div class="author_details">{{ posted_or_edited }} {{ edited_by }}</div></div>
+      </div>
+    </div>
+</div>
+<hr/>
+{{/each}}
```

Comment:
Some one thing I noticed when playing with this frontend is that any newlines in the content of the messages seem to be lost in the edit history displays (and it seems to be showing raw content, not rendered content, which is maybe fine (should probably at least be available as an option) but not what I expected).  Do you know why that is?